reference/content-types/text_editor.rst

```diff
@@ -22,6 +22,21 @@ Parameters
     * - links
       - boolean
       - Adds buttons for creating links to the text editor
+    * - scripts
+      - boolean
+      - Allow script tags in source code
+    * - iframes
+      - boolean
+      - Allow iframe tags in source code
+    * - height
+      - integer
+      - Set the text editor height
+    * - maxHeight
+      - integer
+      - Set the max height for autogrow '0' is  infinitely
+    * - enterMode
+      - string P/BR/DIV
+      - Set what happens on enter allowed P, BR or DIV
```

Comment:
you must put "code words" in double back-ticks. This both helps distinguish them from normal text (for humans) and also for the parser for when we add spell checking.

````
Set what happens, one of: ``p``, ``br`` or ``div``
````